### PR TITLE
Zombie Disease Lethality Buff

### PR DESCRIPTION
**Pull request by Crystalwarrior**
```
 - tweak: "Buffed zombie disease - now it's generally more lethal and causes constant blood drain."
```

Also it's 3 times faster to turn into zombie mob if you're dead.

### CODE REVIEW

**Comment by calibraptor** [comment]
+1

File: code/datums/diseases/zombie.dm [hunk 1]
```diff
@@ -70,29 +72,30 @@ var/list/zombie_cure = list()
...
-					H.vessel.remove_reagent("blood",rand(1,2))
+				else if(prob(5))
+					H.vessel.remove_reagent("blood",rand(5,15))
 					affected_mob.visible_message("<span class='warning'>[affected_mob] looks a bit pale...</span>", "<span class='notice'>You look a bit pale...</span>")
 				else if(prob(5))
 					affected_mob << "<span class='notice'>[pick("You feel hungry.", "You crave for food.")]</span>"
 					affected_mob.overeatduration = max(affected_mob.overeatduration - 25, 0)
 					affected_mob.nutrition = max(affected_mob.nutrition - 25, 0)
 			if(4)
+				var/mob/living/carbon/human/H = affected_mob
```
Comment (JohnGinnane) [inline code/datums/diseases/zombie.dm:83]: I know it's very unlikely for affect_mob to be anything but a human, but is it possible to add a return if !H?

**Comment by JohnGinnane** [comment]
I just one that one minor complaint, but other than that it looks good

**Comment by Sarcalogo** [comment]
May want to close this because of Gandalf's new zombies. Unless this is a modification on his.

**Comment by Crystalwarrior** [comment]
Closed

**Pull request closed without merging**